diff -Naupr snes9x-1.51-src.orig/unix/unix.cpp snes9x-1.51-src/unix/unix.cpp
--- snes9x-1.51-src.orig/unix/unix.cpp	2007-04-29 02:51:08.000000000 +0200
+++ snes9x-1.51-src/unix/unix.cpp	2009-05-06 10:42:48.493984726 +0200
@@ -1162,8 +1162,8 @@ void _splitpath(const char *path, char *
 {
   *drive = 0;
 
-  char *slash = strrchr(path, SLASH_CHAR);
-  char *dot = strrchr(path, '.');
+  const char *slash = strrchr(path, SLASH_CHAR);
+  const char *dot = strrchr(path, '.');
 
   if (dot && slash && dot < slash)
   {
