From 35389f5595d796cbc1fdd527781897133fc62d07 Mon Sep 17 00:00:00 2001
From: OV2 <OV2@sessionclan.de>
Date: Sat, 16 Jun 2018 16:30:38 +0200
Subject: [PATCH] GTK+: use new constant name (#338)

---
 gtk/src/gtk_display.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/src/gtk_display.cpp b/gtk/src/gtk_display.cpp
index 75118ca8..509a3ad1 100644
--- a/gtk/src/gtk_display.cpp
+++ b/gtk/src/gtk_display.cpp
@@ -247,7 +247,7 @@ internal_convert_mask (void         *src_buffer,
                        unsigned int bpp)
 {
 #ifdef __BIG_ENDIAN__
-    if (endianess == ENDIAN_MSB)
+    if (endianess == ENDIAN_SWAPPED)
 #else
     if (endianess == ENDIAN_NORMAL)
 #endif
